web3.js/src/connection.ts

```diff
@@ -3106,6 +3174,55 @@ export class Connection {
     }
   }
 
+  /**
+   * Registers a callback to be invoked whenever logs are emitted.
```

Comment:
Can you please add a comment saying that this API is currently unstable? Or @mvines can you comment on whether the logs pubsub api can be stable now?

Reply:
I'm game for declaring it stable.